src/System.Security.Cryptography.Algorithms/tests/System.Security.Cryptography.Algorithms.Tests.csproj

```diff
@@ -47,6 +53,21 @@
     <Compile Include="$(CommonTestPath)\System\Security\Cryptography\AlgorithmImplementations\AES\AesFactory.cs">
       <Link>CommonTest\AlgorithmImplementations\AES\AesFactory.cs</Link>
     </Compile>
+    <Compile Include="$(CommonTestPath)\System\Security\Cryptography\AlgorithmImplementations\DSA\DSAFactory.cs" Condition="'$(TargetGroup)'=='netstandard1.7'">
```

Comment:
mm I'm not sure if I'm getting your question right, but in theory we shouldn't be changing the netcore50 or netcoreapp1.0 builds at all (this also means adding files) The reason for this is that we have shipped already a stable version for those frameworks, so we would only modify the build for those specific targetGroups in the case where we are bugfixing. Did that more or less answered your question?